packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFWebViewHostApi.m

```diff
@@ -251,7 +253,11 @@ - (void)loadHTMLForWebViewWithIdentifier:(nonnull NSNumber *)identifier
 
 - (void)reloadWebViewWithIdentifier:(nonnull NSNumber *)identifier
                               error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
-  [[self webViewForIdentifier:identifier] reload];
+  if ([self webViewForIdentifier:identifier].URL == NULL && _previousUrlRequest != NULL) {
```

Comment:
`NULL` is not the correct typed value here; a null Obj-C object is `nil`.

Reply:
Fix in https://github.com/flutter/plugins/pull/6887/commits/6e24351fa77718866bcad8f12b982b058d6dc5dd.